### ISSUE TITLE
Visualization flickers when scrolling or typing in other applications

### ISSUE THREAD

**Issue by mtjhax** [radiant-player/radiant-player-mac#521]
When visualization is enabled, scrolling, moving windows, and sometimes even typing in other apps causes the visualization to flicker or flash. Basically, the visualization disappears momentarily, exposing the underlying song list, then reappears.

Radiant Player 1.7.3
OS X Yosemite 10.10.4 on a Macbook Pro (Mid 2014, 15-inch Retina, 2.2 GHz, 16 GB DDR3)

**Comment by jacobwgillespie** [comment]
This is the cover art visualization, right?  Can you see if you can reproduce this in Safari (i.e. open Google Play Music in Safari and see if it happens there).

**Comment by mtjhax** [comment]
Yes it is the cover art visualization. It does not happen with Google Play Music open in Safari. I thought it might be related to my dual monitor setup but I tested it on a single monitor as well with the same result. 

Videos:
https://www.dropbox.com/s/nnhy04o95zlhdtr/radiant%20player.mov?dl=0
https://www.dropbox.com/s/judl426n7gizd2h/google%20play%20music%20in%20safari.mov?dl=0

Obviously not the most urgent of bugs.
